ui/src/services/hooks/api.ts

```diff
@@ -87,9 +87,13 @@ export const useLocalStack = (): useLocalStackReturn => {
     cacheKey,
     async () => (await ddClient.docker.listContainers() as [DockerContainer])
       .find(container =>
-        container.Image === 'localstack/localstack' &&
+        (container.Image.split(':').at(0) === 'localstack/localstack' ||
```

Comment:
Looks similar to the logic further above - see https://github.com/localstack/localstack-docker-extension/pull/25/files#diff-319ece5030e65797e2674d0d24f2ce71fdadcb4a65ab51df55f60f2b1b9f49a7R100 . Maybe we could pull this into a small utility function, and also import the image names from `constants/common.ts`?